content/en/account_management/billing/aws.md

```diff
@@ -25,12 +25,16 @@ You can also limit AWS metrics using the [API][4].
 
 **Note**: Only EC2 (hosts), Lambda (active functions), and CloudWatch Custom Metrics (custom metrics) are billable by Datadog. Metrics integrated for the other services you can filter do not incur Datadog charges.
 
-**Note**: EC2 metrics resource exclusion settings apply to both EC2 and its attached EBS volumes. 
+### EC2
 
-When adding limits to existing AWS accounts within the integration page, the previously discovered instances could stay in the [Infrastructure List][5] up to 2 hours. During the transition period, EC2 instances display a status of `???`. This does not count towards your billing.
+EC2 metrics resource exclusion settings apply to both EC2 instances and any attached EBS volumes. When adding limits to existing AWS accounts within the integration page, the previously discovered instances could stay in the [Infrastructure List][5] for up to two hours. During the transition period, EC2 instances display a status of `???`. This does not count towards your billing.
 
 Hosts with a running Agent still display and are included in billing. Use the limit option to restrict `aws.ec2.*` metrics collection from AWS and restrict the AWS resource EC2 instance hosts.
 
+### CloudWatch Metric Streams with Kinesis Data Firehose
+
+You can optionally [forward CloudWatch Metric Streams to Datadog with the Kinesis Data Firehose][8]. If you use this method, you must manage all rules for including and excluding namespaces in the streams using the CloudWatch Metric Streams configuration in your AWS accounts. Namespace defaults and account-level settings in the Datadog AWS integration page only apply to the default API polling approach. 
```

Comment:
```suggestion
You can optionally [send CloudWatch metrics to Datadog using CloudWatch Metric Streams and Kinesis Data Firehose][8] instead of with polling. If you use this method, AWS resource exclusion rules do not apply and you must manage all rules for including and excluding metric namespaces or specific metric names in the CloudWatch Metric Streams configuration in your AWS accounts.
```